client/payment-methods-map.tsx

```diff
@@ -247,6 +248,37 @@ const PaymentMethodInformationObject: Record<
 			'woocommerce-payments'
 		),
 	},
+	klarna: {
+		id: 'klarna',
+		label: __( 'Klarna', 'woocommerce-payments' ),
+		brandTitles: {
+			affirm: __( 'Klarna', 'woocommerce-payments' ),
+		},
+		description: __(
+			// translators: %s is the store currency.
+			// TODO: Is "Available to all customers paying in %s." correct for BNPLs given the introduction of domestic check for PMs?.
```

Comment:
IDK, WDYT @dmvrtx?

Reply:
I'd change it to something like "Available for orders placed in the following currency: %s".

Reply:
@KarlisJ: did we clarify the TODO here?

Reply:
+1. Agree with moving it into a separate issue.

Reply:
I looked more into the wording here and it actually seems to be intentional and correct as we allow only domestic payments.